evm-config.schema.json

```diff
@@ -20,6 +20,11 @@
         "chrome"
       ]
     },
+    "preserveXcode": {
```

Comment:
Maybe make it a number, and then preserve the `N` most recent `atime`'ed Xcode folders 🤔